integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitSpecificIntegrationTestBase.java

```diff
@@ -1056,7 +1056,7 @@ public void get_GetGivenForDeletedWhenCoordinatorStateNotExistAndExpired_ShouldA
   }
 
   @Test
-  public void getAndScan_CommitHappenedInBetween_ShouldReadRepeatably()
+  public void getThenScanAndGet_CommitHappenedInBetween_OnlyGetShouldReadRepeatably()
```

Comment:
This is one of the user-facing specification changes by this PR. We do not provide a snapshot across different operations. It's a bit regrettable, though.